demo/index.html

```diff
@@ -29,4 +99,38 @@
 
   <!-- Demo -->
     <script src="annotator-content-analysis_demo.js"></script>
+
+  <!-- temp -->
+    <script>
+      // this is hacky and belongs somewhere else
+      var currentTopic = "1";
+      function collapse(elem) {
+        $(elem).parent().children().toggle();
+        $(elem).toggle()
+               .toggleClass('variable-list__collapsible--closed');
+      }
+      function activateTopic(topic) {
+        $('.topic-picker__nav li').removeClass('active');
+        $(`.topic-picker__nav li[data-topic="${topic}"]`).addClass('active');
+        $('.text-wrapper__text').attr('data-topic', topic);
+
+        $('.topic-wrapper').hide();
+        $(`.topic-wrapper[data-topic="${topic}"]`).show();
+      }
+
+      $(document).ready(_ => {
```

Comment:
Oh, my bad - it's a convention I like borrowed from Python, to mean this variable is not actually used. 
```
for _ in range(5):
    print("hi")
```